tests/phpunit/tests/theme/wpTheme.php

```diff
@@ -275,6 +275,60 @@ public function test_is_block_theme( $theme_dir, $expected ) {
 		$this->assertSame( $expected, $theme->is_block_theme() );
 	}
 
+	/**
+	 * @dataProvider data_is_block_theme
+	 * @ticket 57114
+	 *
+	 * @covers WP_Theme::is_block_theme
```

Comment:
```suggestion
	 * @ticket 57114
	 *
	 * @covers WP_Theme::is_block_theme
	 *
	 * @dataProvider data_is_block_theme
```

Per [56070](https://core.trac.wordpress.org/ticket/56070) proposal the format needs to change.